src/events/guildMemberAdd.js

```diff
@@ -1,14 +1,14 @@
 const { MessageAttachment } = require('discord.js');
 const client = require('..');
-const config = require('../../config.json');
+const config = require('../../config.exemple.json');
```

Comment:
As announced on Discord, config.example.json is a convention for future and current contributors to know the template of the configuration file.
When they work on it and in production, the file should be duplicated and renamed config.json locally, without committing it.

```suggestion
const config = require('../../config.json');
```

Reply:
ah yes my bad
I fix that